### ISSUE TITLE
add AGPL

### ISSUE THREAD

**Issue by nerdoc** [pydanny/cookiecutter-django#2941]
Please add the AGPL license to the list of available licenses! Thanks

**Comment by Andrew-Chen-Wang** [comment]
@browniebroke It might just be worth it to use GitHub's `choosealicense.com`. We could vendor all the licenses from here (in case GitHub changes where they are stored or how they're laid out) and update anytime there is a change in their directory: https://github.com/github/choosealicense.com/tree/gh-pages/_licenses

**Comment by Andrew-Chen-Wang** [comment]
Or instead perhaps make separate directories for each license rather than storing everything in a single file?

**Comment by browniebroke** [comment]
Interesting idea to get them from that repo. A couple of things I noticed:

- the files content are a bit more than just the license, they are templates with some sort of front matter
- templates variables (year and author) seems to be using square brackets while cookiecutter (jinja) uses double curly braces

We could pull them and convert them regularly with a github workflow, but it might not be worth the trouble... 

Splitting the current file into multiple ones would be a nice improvement on the maintenance front, would make it easier to see what we support.

**Comment by Andrew-Chen-Wang** [comment]
Agreed; too much trouble. On the second thought, having to parse for a bunch of dashed lines to indicate the beginning of the license file is not really that good of an idea. But definitely splitting the licenses is an improvement, and importing each license from that repo and replacing them with the jinja curly braces would be needed.
